api/pkg/apis/v1alpha1/managers/activations/activations-cleanup-manager.go

```diff
@@ -83,7 +84,7 @@ func (s *ActivationsCleanupManager) Poll() []error {
 			ret = append(ret, err)
 		}
 		duration := time.Since(updateTime)
-		if duration > time.Duration(s.RetentionInMinutes)*time.Minute {
+		if duration > s.RetentionDuration {
```

Comment:
Where's the check for enabled flag?

Reply:
Enable flag is only used in helm chart now. Do you want standalone mode also able to disable activation cleanup?